src/devices/bus/sega8/sega8_slot.h

```diff
@@ -101,6 +104,8 @@ class device_sega8_cart_interface : public device_interface
 
 	int m_lphaser_xoffs;
 	int m_sms_mode;
+
+	sega8_cart_slot_device *m_slot;
```

Comment:
If you only set this on construction, you should make it `const` and set it in the initialiser list.

Reply:
The changes I sent today removes it.